libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationService.java

```diff
@@ -298,7 +298,7 @@ public void rerouteOccurred() {
       .routeProgressBeforeReroute(routeProgress)
       .beforeRerouteLocations(Arrays.asList(
         locationBuffer.toArray(new Location[locationBuffer.size()])))
-      .lastRerouteDate(new Date())
+      .rerouteDate(new Date())
```

Comment:
Would it be easier just to compute and store `secondsSinceLastReroute` here rather than try to generate the value after the fact right before sending out?